packages/ses/test/test-import-non-esm.js

```diff
@@ -1,4 +1,5 @@
 import test from 'ava';
+import { StaticModuleRecord } from '@endo/static-module-record';
```

Comment:
What's the difference between this and the one that was previously available as a global?

Reply:
Long ago, we moved `StaticModuleRecord` out of SES because it entrained too much of Babel. This code predates that migration and had to be brought back into the modern era. The proposal to TC39 includes `StaticModuleRecord` as a global, so this move is a reduction in the fidelity of the shim.